_layouts/base.html

```diff
@@ -85,7 +85,7 @@ <h3>Community</h2>
     </div>
 
     <div style='position:relative; margin-top:-2px; padding:50px 0 0 0; background:#fff;'>
-      <p style='background:#5bc8df; color:#fff; text-align:center; padding:10px; margin:0;'>&copy; ScaleConf 2018</p>
+      <p style='background:#5bc8df; color:#fff; text-align:center; padding:10px; margin:0;'>&copy; ScaleConf 2019</p>
```

Comment:
use `{{ 'now' | date: "%Y" }}` instead of 2019 and you'll never need to update it, ever again.